keps/sig-cloud-provider/2436-controller-manager-leader-migration/README.md

```diff
@@ -305,7 +307,8 @@ The default migration configuration is implemented and tested.
 
 ##### Beta -> GA Graduation
 
-Leader migration configuration works on all in-tree cloud providers.
+- Leader Migration works on all in-tree cloud providers that require migration.
+- Leader Migration passes an upgrade test of a replicated control plane.
```

Comment:
This should be extended to specify that we will have an automated tests that HA upgrade from KCM (only) to KCM + CCM and downgrade from KCM + CCM to KCM (only) complete successfully with the feature controller manager leader migration enabled.

Reply:
Added.